template/src/containers/App/Router.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
We should stick to either single quotes or double quotes. Personally I prefer singles, but as long as they are all the same I am fine either way

Reply:
🤦, that's why we need prettier.